Doc/library/sqlite3.rst

```diff
@@ -1782,8 +1782,8 @@ case-insensitively by name:
 
 .. _sqlite3-connection-context-manager:
 
-Using the connection as a context manager
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+How to use a connection as a context manager
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Should we use a more generic plural form?
```suggestion
How to use connections as a context manager
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
Is there a reason for making it plural? I'm not sure I grasp the rationale here; the indefinite singular is the closest in line with the section text, though suggested rephrasing by @erlend-aasland avoids the conflict. I'm +0, maybe +0.5 on it over the current as its a touch more concise and specific; here it is as an actual suggestion:

```suggestion
How to use the connection context manager
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
I really like that last suggestion, CAM.